plugins/org.eclipse.n4js.npmexporter/src/org/eclipse/n4js/npmexporter/ConvertManifestToPackageJson.xtend

```diff
@@ -0,0 +1,81 @@
+/**
+ * Copyright (c) 2016 NumberFour AG.
+ * All rights reserved. This program and the accompanying materials
+ * are made available under the terms of the Eclipse Public License v1.0
+ * which accompanies this distribution, and is available at
+ * http://www.eclipse.org/legal/epl-v10.html
+ *
+ * Contributors:
+ *   NumberFour AG - Initial API and implementation
+ */
+package org.eclipse.n4js.npmexporter
+
+import org.eclipse.n4js.n4mf.ProjectType
+import org.eclipse.n4js.projectModel.IN4JSProject
+import org.eclipse.n4js.runner.^extension.RuntimeEnvironment
+import java.nio.file.Path
+
+import org.eclipse.n4js.semver.model.SemverSerializer
+
+// GH-809 delete this class
+/**
+ * Converting all relevant Manifest-data into an DataObject.
+ */
+class ConvertManifestToPackageJson {
```

Comment:
This class was deleted as part of the `GH-809` clean up. Now it seems you re-create this class.